README.md

```diff
@@ -41,9 +42,13 @@
    - All pages that load correctly should respond with a 2xx status code
    - Not found (404) pages should respond with a 404 status code
    - Any API should also send correct status codes
+7. not be fully filled with ads and/or paid placeholders.
+8. have API (and the frontend if possible) ratelimits. (60 requests per minute recommended)
+9. have a support (email and/or discord server) link on navbar or footer.
+10. notify users beforehand when your tos or privacy pages are changing.
 
 ---
 
-If anyone wants to change or add something, please [make a pull request](https://github.com/botblock/discord-botlist-best-practices).
+If anyone wants to change or add something, please make a pull request [here](https://github.com/botblock/discord-botlist-best-practices).
```

Comment:
Don't see a point in changing this.

Reply:
"make a pull req" would mean "hey visitor, goto github and make a pull req there", whereas adding "here" may give a clue to user that "here" means that link. I know links are pointed out blue, but I have seen a lot of sources and readmes/docs where they say "here"(and link it)

Reply:
Using "here" for links is a UX anti-pattern, the link should provide context to what it is linking the user to, which it did before.

Reply:
> "make a pull req" would mean "hey visitor, goto github and make a pull req there", whereas adding "here" may give a clue to user that "here" means that link. I know links are pointed out blue, but I have seen a lot of sources and readmes/docs where they say "here"(and link it)

I don't get your logic here.
When it was saying `please [make a pull request](link).` it was more than clear that the link refers to the GitHub Repository because... what else would it mean?